src/training/predict.py

```diff
@@ -0,0 +1,85 @@
+import math
+import numpy as np
+import tifffile as tiff
+import os
+
+from train_unet import weights_path, get_model, normalize, PATCH_SZ, N_CLASSES
+
+if not os.path.exists('data/WV_predicted'):
+        os.makedirs('data/WV_predicted')
+
+image= normalize(tiff.imread('name of the WV image.tif').transpose([1,2,0]))
+wind_row, wind_col = 800,800 # dimensions of the image
+windowSize = 800 
+stepSize=400
+
+
+desired_size_row=stepSize*math.ceil(image.shape[0]/stepSize)
+desired_size_col=stepSize*math.ceil(image.shape[1]/stepSize)
+
+img = np.zeros((desired_size_row,desired_size_col,image.shape[2]), dtype=image.dtype)
+
+img[:image.shape[0],:image.shape[1]] = image
+

```

Comment:
Are the variables in lines 11 to 22 needed as global? If they are only used from the main, they can be moved there! The same is true with lines 8-9.